src/widgets/TOCWidget.vue

```diff
@@ -36,17 +53,34 @@
       return {
         toc: null,
         filtered: null,
+        showURNs: false,
       };
     },
     computed: {
+      context() {
+        return this.$route.name;
+      },
+      placeholder() {
+        return 'Search this table of contents...';
+      },
+      passage() {
+        return this.$store.getters[`${WIDGETS_NS}/passage`];
+      },
       metadata() {
         return this.$store.getters[`${WIDGETS_NS}/metadata`];
       },
       reducer() {
         return reducers.tocReducer;
       },
-      placeholder() {
-        return 'Search this table of contents...';
+      showingRootToc() {
+        return this.context == 'tocs'
+          ? !this.$route.query.urn
+          : !this.$route.query.toc;
```

Comment:
@jhrr consider this use case:

1) Browse to https://deploy-preview-13--sv-mini.netlify.com/reader?urn=urn%3Acts%3AgreekLit%3Atlg1443.tlg004.oaf-1%3A0-1
2) We show the "default" toc for OAF
3) We cannot navigate back to "root":

![image](https://user-images.githubusercontent.com/629062/76347157-2285bc00-62d4-11ea-90ef-4445516b78cb.png)

Maybe add to the conditional in `showingRootToc` to detect if the root toc (vs default TOC) is being shown?

Reply:
Ah yes! Great catch, will fix that up.